sys/include/net/gnrc/ipv6/nc.h

```diff
@@ -149,6 +149,9 @@ typedef struct {
     xtimer_t nbr_adv_timer;
     msg_t nbr_adv_msg;                          /**< msg_t for gnrc_ipv6_nc_t::nbr_adv_timer */
 
+    xtimer_t rtr_adv_timer;                     /**< Timer for periodic router advertisements */
```

Comment:
Can you please add `#ifdef defined(MODULE_GNRC_NDP_ROUTER) || defined(MODULE_GNRC_SIXLOWPAN_ND_BORDER_ROUTER)` around all newly introduced lines?